druid/src/widget/image.rs

```diff
@@ -58,6 +60,17 @@ impl<T: Data> Image<T> {
     pub fn set_fill(&mut self, newfil: FillStrat) {
         self.fill = newfil;
     }
+
+    /// A builder-style method for specifying the interpolation strategy.
+    pub fn interpolation_mode(mut self, interpolation: InterpolationMode) -> Self {
+        self.interpolation = interpolation;
+        self
+    }
+
+    /// Modify the widget's `InterpolationMode`.
+    pub fn set_interpolation(&mut self, interpolation: InterpolationMode) {
```

Comment:
I'd like to give these consistent names, even if it's long, so this would be `set_interpolation_mode`.

Reply:
The fill has mode for the builder and set for the updater. I think that's what you wanted so i just did the same but changed fill to interpolation.

I can change both..

Reply:
what I mean is that I think the name should be `interpolation_mode` and `set_interpolation_mode`; they should match, just one has  `set`.

Reply:
I mean that we have
```
    pub fn fill_mode(mut self, mode: FillStrat) -> Self
    pub fn set_fill(&mut self, newfil: FillStrat)
```
so to be consistent we would have
```
    pub fn interpolation_mode(mut self, interpolation: InterpolationMode) -> Self 
    pub fn set_interpolation(&mut self, interpolation: InterpolationMode) 
```
But do we want `set_fill` to be `set_fill_mode` so we have

```
    pub fn fill_mode(mut self, mode: FillStrat) -> Self
    pub fn set_fill_mode(&mut self, newfil: FillStrat)
    pub fn interpolation_mode(mut self, interpolation: InterpolationMode) -> Self 
    pub fn set_interpolation_mode(&mut self, interpolation: InterpolationMode) 
```

Reply:
ah gotcha. yea, that's an oversight, we can fix that in another patch? Let's do these 'right', and then we can patch up the other one. :)